model/src/signature.rs

```diff
@@ -8,7 +8,7 @@ use web3::{
     types::Recovery,
 };
 
-#[derive(Eq, PartialEq, Clone, Copy, Debug, Deserialize, Serialize, Hash)]
+#[derive(Eq, PartialEq, Clone, Copy, Debug, Serialize, Hash)]
```

Comment:
I'm not sure I agree with this change. I would, instead of removing the `Deserialize` add a test for deserialization. My rational being that the `model` crate can be used by some other Rust project that wants to deserialize API responses from our orderbook service.

Reply:
That cool I can put it back, but shouldn't we only include this when it is actually used?

Reply:
Yeah, I guess you're right. We can always add it back if needed.

Reply:
and its derived, so it doesn't require any effort to add it back.